### PR TITLE
Pin Apache Arrow version

### PR DESCRIPTION
**Pull request by lava**

### CODE REVIEW

**Comment by mavam** [comment]
```
vast: error while loading shared libraries: libarrow.so.100: cannot open shared object file: No such file or directory
2244
```

**Review by tobim — CHANGES_REQUESTED** [review]
You need to update `github/workflows/jupyter.yaml` and `Dockerfile_prebuilt` too.

**Comment by mavam** [comment]
@tenzir/backend IIRC we discussed adding a document on what to do when touching dependencies, because it's impossible that everyone remembers all the different places in the repo.

File: Dockerfile_prebuilt [hunk 1]
```diff
@@ -7,10 +7,12 @@ RUN apt-get -qq update && apt-get -qq install -y libasan5 libc++1 libc++abi1 \
...
-RUN wget https://apache.bintray.com/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb && \
-  apt-get -qqy install ./apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb && \
-  apt-get -qq update && \
-  apt-get -qqy install libarrow-dev
+# TODO: Revert the commit that introduced this version pinning once ch20162
+# is done and we can use latest upstream again.
+RUN wget https://apache.bintray.com/arrow/ubuntu/pool/focal/main/a/apache-arrow-archive-keyring/apache-arrow-archive-keyring_1.0.1-1_all.deb && \
+  sudo apt-get -qqy install ./apache-arrow-archive-keyring_1.0.1-1_all.deb && \
+  sudo apt-get -qq update && \
+  sudo apt-get -qqy install libarrow-dev=1.0.1-1
```
Comment (tobim) [inline Dockerfile_prebuilt:15]: No sudo needed.

**Review by tobim — CHANGES_REQUESTED** [review]

**Comment by tobim** [comment]
`vast: error while loading shared libraries: libarrow.so.17: cannot open shared object file: No such file or directory`
in the jupyter tests.

**Comment by tobim** [comment]
The runs of the jupyter workflow used vast-2020.06.25-26-gda6d112d, which is quite old. Is it set up like that intentionally? (@0ortmann)

**Review by tobim — APPROVED** [review]

**Pull request merged**